src/main/java/platform/codingnomads/co/springdata/lab/models/Route.java

```diff
@@ -21,22 +21,29 @@ public class Route implements Serializable {
     @GeneratedValue(strategy = GenerationType.AUTO)
     private Long id;
 
-    @Column(unique = true)
-    private String code;
-
-    @ManyToOne(fetch = FetchType.LAZY, optional = false)
+    @ManyToOne(fetch = FetchType.EAGER, optional = false) // Had to change these to EAGER to fix a session issue. Is there a better way?
```

Comment:
I can't recall what the problem was, is it still present? Do you have the error output?

Reply:
It was a session closed error. Basically the error was saying that the session had already closed when it was trying to use toString(). I tried to solve it with @Transactional at the class level first, but that didn't seem to do anything.